webui/react/src/services/utils.ts

```diff
@@ -40,9 +41,21 @@ export const isNotFound = (e: any): boolean => {
   return getResponseStatus(e) === 404;
 };
 
+// DISCUSS Let's not hide this shameful act and directly use globalStorage.
+// export const getAuthToken = (): string => {
+//   const token = globalStorage.get('auth-token');
+//   console.log('auth token', !!token);
+//   return (token as string);
+// };
+// export const setAuthToken = (token: string): void => {
+//   // DISCUSS we could set this up through react contexts but then all usage of APIs
+//   // would have to be moved to follow suit.
+//   globalStorage.set('auth-token', token);
+// };
+
 /* HTTP Helpers */
 
-export const http = axios.create({ responseType: 'json', withCredentials: true });
+export const http = axios.create({ responseType: 'json', withCredentials: false });
```

Comment:
**chore:** We can just drop `withCredentials` here since it will default to false.

Reply:
I agree I did that in another coomit. I think you're going commit by commit. I would have advised against that. I went back a forth many times in this PR.